app/Http/Controllers/MainController.php

```diff
@@ -38,16 +38,25 @@ public function place(){
         return view('View_place', ['cities_names' => $cities_names,'stay'=> $stay,'place_name'=> $place_name]);
 
     }
-    public function getPlace(Request $request)
+    public function getPlaces(Request $request)
     {
         $id_req = $request->city_id;
         $places = Place::all()->where('id_city', $id_req);
         return array_map(function($place){
             return array ('id' => $place->id, 'text' => $place->address);
         },$places->all());
     }
+    public function getPlaces2(Request $request)
```

Comment:
What do you mean 'getPlaces2' ?

Reply:
It is templayte. because backend is not ready yet.

Reply:
Ok, then add some comments in the code.